app/admin/governance/multitenant_wallet/wallet_admin.py

```diff
@@ -0,0 +1,428 @@
+import json
```

Comment:
Not sure whether I added this, but I don't think we need this? The wallet groups allows you to assign a group to a wallet. we already have the tenants api that can incorporate these groups into the api. I don't think we want to expose the multitenancy api as exposed in ACA-Py directly

Reply:
Agreed. Sounds good to me. Not sure who added it but it was already in the PR - but matters not who did it. I'll address this within the tenants api then.

Reply:
If it was already there it was me! Well good to be critical of my own work

Reply:
@TimoGlastra moved everything into tenants and corresponding logic